README.md

```diff
@@ -101,24 +103,18 @@ Oh, but don't say the "F" word unless the client does first, please 😜
 
 ## What has worked for us in the past
 
-We've found the following things work really well working with clients:
+[Establishing trust with clients](soapbox/people/afraser.md) can make the difference between getting the job done and exceeding expectations, [afraser](https://github.com/afraser) has some opinions on this stuff, but here's a high-level:
 
 - Communicate. A lot.
-- Get shit done.
-- Speak from experience.
-  - Explain how things made a team or users happy or unhappy.
-  - There is no "best practice", there's only practice.
-- Present ideas frequently.
-  - Get decision makers on your side and/or find a [wolf pack](https://www.urbandictionary.com/define.php?term=Wolfpack)
-- Build it, then show and tell.
-  - New ideas are much more likely to get pulled in if they're already "mostly done."
-- Make developers' lives easier.
-  - Automate things. Improve performance. Find pain points and fix them.
-- Transcend your team.
-  - Communicate and become best friends with Design, UX, Product Owners, Architecture, DevOps... Etc.
-- Pay attention to detail.
-  - Don’t get sloppy (slow code, bad naming, missed edge cases, etc.) .
-  - Delight developers, designers and users.
+- Set expectations. Deliver on promises.
+- Speak in terms of what makes people happy or unhappy.
+- Big ideas are more likely to be entertained if they're already built. Build it, then show & tell.
+- Make developers' lives easier through automation etc.
+- Transcend your team by communicating with Design, UX, etc.
+- Speak in terms of real users and use cases.
+- Pay attention to detail. Delight developers, designers and users.
+- Know when and how much to push back.
```

Comment:
I like this.  But what about:

`Know when and how much to push back. Pick your battles and always _be tactful and respectful_.`

Reply:
I feel like the second bit is redundant (pick your battles is the "when", I'll add the bit about tact to the soapbox though)